### PR TITLE
Set up a "Hello Microverse" projectFeature

### PR DESCRIPTION
**Pull request by odaialazzeh**
- Add a new GitHub repo.
- Add a new branch 
- Add a .gitignore file.
- Check the linters.
- Add a descriptive README file to your project.
- Add a .html file that includes a header with the text "Hello Microverse!". Use the h1 tag for that.
- Add a .css file that includes one class with styles for your header (make it in your favorite color).
- Add a test.md file with the text "This file should be ignored by git".
- Ignored the last file in the repository.
- Open a pull request.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/test.md
+
+/node_modules
+
+/*.json
```
Comment (franclobo) [inline .gitignore]: - [ ] Kindly remove `/*.json` from `.gitignore` file. It is not part of the requirements.
- [ ] Please correct the name of `/node_modules` by `node_modules/`

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @odaialazzeh, 

Good job so far!
You are doing it well 👍🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive PR ✔️ 
- Descriptive  commits ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Kindly add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
- [ ] Please 🙏🏽 add a copy of [.github/workflows/linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml) to check the linters.
image

- [ ] Kindly remove the `index.json` file from your repository it is not part of this project. On the contrary, `package.json` is automatically created when you run `npm init -y`.
- [ ] Please 🙏🏽 add the MIT LICENSE file to your repository. We could follow these [steps](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository).

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @odaialazzeh  , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to resolve the conflicts between your files. You can refer to this [video ](https://www.youtube.com/watch?v=JtIX3HJKwfo)and try to follow all steps to resolve your conflicts successfully. :wink: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Kindly make sure to write your commit message in imperative tense and capitalized it. Please check this [lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to understand it clearly.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — COMMENTED** [review]
Hi @odaialazzeh, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 
- [ ] Has broken link.
- [ ] The feature branch is not accessible.
- [ ]  This is what I got while trying to open your feature branch.





Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by awais-amjed — COMMENTED** [review]
# STATUS: APPROVED 🥳

Hi @odaialazzeh , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations on implementing the required changes! 🎉👏🥇✨

- [ ] _**Please do not merge or delete a branch before approval from a Code Reviewer**, kindly keep this in mind for the future. Since you are practicing GitHub flow in this project, so I will be approving your project as you have already implemented all the changes required by the previous reviewers. Just be careful in the future and happy coding 🤗_

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] No Linter issues ✔️
- [x] Good code organization ✔️ 

gif

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**